components/brave_ads/browser/ads_p2a.h

```diff
@@ -7,19 +7,89 @@
 #define BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_P2A_H_
 
 #include <string>
+#include <vector>
 
 class PrefService;
 class PrefRegistrySimple;
 
 namespace brave_ads {
 
+const std::vector<std::string> kP2AQuestionList = {
```

Comment:
non-trivial global vars are prohibited, could we replace it with plaing c-style array and char[] ?